backend/YouTubeV2.sln

```diff
@@ -0,0 +1,53 @@
+﻿
```

Comment:
isn't the application set to startup project? if it is change it if not maybe it's just like this on my computer

Reply:
application is a library so it cannot be a startup project. everything works fine on my end.